pom.xml

```diff
@@ -63,14 +63,14 @@ THE SOFTWARE.
         </pluginRepository>
     </pluginRepositories>
     <properties>
-        <jenkins.version>1.642.3</jenkins.version>
-        <scm-api.version>2.0.7</scm-api.version>
+        <java.level>8</java.level>
+        <jenkins.version>2.62</jenkins.version>
```

Comment:
Why not 2.73.3 BTW?

Reply:
Minimum core version that supports the features we need, so maximum number of users can apply it. 

If I'd caught it, I would have blocked the upstream update that mandates core 2.62 as well until after the release of this -- a trivial cleanup isn't worth forcing people to a much later LTS, unless we're doing it anyway.